### PR TITLE
v6.19.4

### PR DESCRIPTION
**Pull request by DanailH**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,62 @@
...
+- [charts] Allows series ids to be numbers (#11941) (#11960) @alexfauquette
+- [charts] Fix Proptypes by support string values for axes (#11953) @alexfauquette
+
+### Docs
+
+- [docs] Add a note about `AdapterDateFnsV3` on the Getting Started page (@flaviendelangle) (#11987) @github-actions[bot]
+- [docs] Avoid the use of MUI Core @oliviertassinari
+- [docs] Fix API links (#11930) @alexfauquette
+- [docs] Fix `ChartsTooltip` typo (@thisisharsh7) (#11967) @github-actions[bot]
+- [docs] Refactor `Localization` documentation sections (@LukasTy) (#11997) @github-actions[bot]
```
Comment (LukasTy) [inline CHANGELOG.md:59]:
```suggestion
- [docs] Refactor `Localization` documentation sections (#11997) @LukasTy
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,62 @@
...
+- [code] Simplify bug reproduction (#11849) (#11932) @alexfauquette
```
Comment (LukasTy) [inline CHANGELOG.md:60]:
```suggestion
- [code] Simplify bug reproduction (#11932) @alexfauquette
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,62 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.19.4
+
+_Feb 9, 2024_
+
+We'd like to offer a big thanks to the 4 contributors who made this release possible. Here are some highlights ✨:
```
Comment (LukasTy) [inline CHANGELOG.md:10]:
```suggestion
We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,62 @@
...
+
+- 🌍 Improve  Danish (da-DK) locale on the Data Grid (#11831) @danielmishan85
```
Comment (LukasTy) [inline CHANGELOG.md:12]:
```suggestion
- 🌍 Improve  Danish (da-DK) locale on the Data Grid (#11972) @ShahrazH
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,62 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.19.4
+
+_Feb 9, 2024_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve  Danish (da-DK) locale on the Data Grid (#11972) @ShahrazH
```
Comment (LukasTy) [inline CHANGELOG.md:12]:
```suggestion
- 🌍 Improve Danish (da-DK) locale on the Data Grid (#11972) @ShahrazH
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,62 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.19.4
+
+_Feb 9, 2024_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
```
Comment (LukasTy) [inline CHANGELOG.md:10]: Sorry for miscalculating initially. 🙈 
```suggestion
We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
```

**Review by LukasTy — APPROVED** [review]
🚀 
P.S. It would be nice to include the changelog in v7 release notes. 😉

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,62 @@
...
+- [pickers] Avoid relying on locale in Luxon `isWithinRange` method (#11940) @LukasTy
+
+#### `@mui/x-date-pickers-pro@v6.19.4` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@v6.19.4`.
+
+### Charts / `@mui/x-charts@v6.19.4`
+
+- [charts] Add `reverse` property to axes (#11959) @alexfauquette
+- [charts] Allows series ids to be numbers (#11960) @alexfauquette
```
Comment (alexfauquette) [inline CHANGELOG.md:50]:
```suggestion
- [charts] Allow series ids to be numbers (#11960) @alexfauquette
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,62 @@
...
+- [charts] Fix Proptypes by support string values for axes (#11953) @alexfauquette
```
Comment (alexfauquette) [inline CHANGELOG.md:51]:
```suggestion
- [charts] Fix Proptypes error by supporting string values for axes (#11953) @alexfauquette
```

**Review by alexfauquette — APPROVED** [review]

**Comment by DanailH** [comment]
> 🚀 P.S. It would be nice to include the changelog in v7 release notes. 😉

I don't have an issue doing it but it seems that it hasn't been done before?

**Comment by flaviendelangle** [comment]
> I don't have an issue doing it but it seems that it hasn't been done before?

Maybe we forgot some time, but we tried to always do the v6 => v7 sync so that the CHANGELOG in v7 always contain all the past releases and we don't loose the releases of v6 done after the first v7 release

**Comment by DanailH** [comment]
@flaviendelangle I see, they are not ordered by date in the CHANGELOG. That's why I missed it. I've included the changes in the v7 changelog

**Pull request merged**